### PR TITLE
Add cmd UI accessibility feature to tab through screen items

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: integration-tests/integration/calendar-overtime.spec.js [hunk 1]
```diff
@@ -15,50 +14,67 @@ context('A staff member can view their overtime calendar', () => {
     cy.task('stubNotificationCount')
     cy.login()
 
-    utilities.gotoPreviousCalendarDate('2020-03-01')
+    CalendarPage.verifyOnPage(moment().format('MMMM YYYY'))
+    cy.visit('/calendar/2020-04-01')
+    cy.get('[data-qa=previous]').click()
```
Comment (steverendell) [inline integration-tests/integration/calendar-overtime.spec.js:19]: gotoPreviousCalendarDate() was intended to get us from the current month view to a specified month which is so long ago it now takes ages. now simplified to just go to a month or 2 later then go back

File: integration-tests/integration/calendar-overtime.spec.js [hunk 2]
```diff
@@ -15,50 +14,67 @@ context('A staff member can view their overtime calendar', () => {
...
+      .within(() => {
+        cy.get('span').eq(0).contains('Saturday 7th')
+        cy.get('span').eq(1).contains('7')
+        cy.get('span').eq(2).contains('Start 07:30')
+        cy.get('span').eq(3).contains('Finish 12:30')
+        cy.get('span').eq(4).contains('5hrs')
+        cy.get('span').eq(5).contains('Start 12:30')
+        cy.get('span').eq(6).contains('Finish 21:00')
+        cy.get('span').eq(7).contains('8hrs')
+      })
```
Comment (steverendell) [inline integration-tests/integration/calendar-overtime.spec.js:36]: some of the spans are now within a div

File: server/views/pages/calendar.ejs [hunk 3]
```diff
@@ -28,8 +28,8 @@
         </div>
```
Comment (steverendell) [inline server/views/pages/calendar.ejs:29]: added some tabindex-es so each date number is visited followed by the contents div

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**